data/tooltips.txt

```diff
@@ -164,6 +167,9 @@ tip "mass with no cargo:"
 tip "mass:"
 	`Mass, in tons. A ship's mass determines how fast it turns and accelerates.`
 
+tip "operating costs:"
+	`How many credits per day you need to spend to maintain this outfit while it is in active use.`
```

Comment:
Again, not just outfits. This attribute can also apply to ship hulls.

Reply:
As with above, this tooltip is only visible on outfits, hence why the tooltip mentions outfits.

Reply:
Ok